Sources/KeyboardShortcuts/Localization/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,6 @@
+"Record Shortcut" = "Record Shortcut";
+"Press Shortcut" = "Press Shortcut";
+"This keyboard shortcut cannot be used as it's already used by the" = "This keyboard shortcut cannot be used as it's already used by the";
+"menu item." = "menu item.";
+"This keyboard shortcut cannot be used as it's already a system-wide keyboard shortcut." = "This keyboard shortcut cannot be used as it's already a system-wide keyboard shortcut.";
+"Most system-wide keyboard shortcuts can be changed in “System Preferences › Keyboard › Shortcuts“." = "Most system-wide keyboard shortcuts can be changed in “System Preferences › Keyboard › Shortcuts“.";
```

Comment:
Do we really need to have a file that just duplicates the source?

Reply:
This is my coding habit, which makes file structure more clear.

Reply:
But they can just look at any other translation and look at the source strings there.

Reply:
Deleted